base/sparse/sparsematrix.jl

```diff
@@ -1212,6 +1212,32 @@ sprandn(m::Integer, n::Integer, density::AbstractFloat) = sprandn(GLOBAL_RNG,m,n
 
 Create a sparse array with the same structure as that of `S`, but with every nonzero
 element having the value `1.0`.
+
+```jldoctest
+julia> A = sprand(5,6,0.2)
+5×6 sparse matrix with 8 Float64 nonzero entries:
+    [2, 1]  =  0.639431
+    [5, 1]  =  0.881209
+    [3, 2]  =  0.355834
+    [4, 2]  =  0.904768
+    [2, 3]  =  0.760943
+    [3, 5]  =  0.525942
+    [4, 5]  =  0.936283
+    [5, 5]  =  0.432364
+
+julia> spones(A)
+5×6 sparse matrix with 8 Float64 nonzero entries:
+    [2, 1]  =  1.0
+    [5, 1]  =  1.0
+    [3, 2]  =  1.0
+    [4, 2]  =  1.0
+    [2, 3]  =  1.0
+    [3, 5]  =  1.0
+    [4, 5]  =  1.0
+    [5, 5]  =  1.0
+```
+
+Note the difference from [`speye`](:func:`speye`).
```

Comment:
while I have made this mistake before, I don't think we say "note the difference from `eye`" in the `ones` docstring, do we?